DTS-STN/Alpha-Site
# Fix tailwind config error

# Description

[https://jira-dev.bdm-dev.dts-stn.com/browse/SCL-552](Tailwind config error)

A regex in tailwind config broke the font and some css. This PR fixes those problems.

## Acceptance Criteria

Font and broken css are fixed.

## Test Instructions

1. Load the home page
2. Notice the font is back to the right one
3. Go to the about page
4. Notice the blue circles now have the correct transparency again.

## Checklist


## Product and Sprint Backlog

[Trello](https://trello.com/b/ZqWtJSyh/alpha-site-board)
